### ISSUE TITLE
Verifier not accepting CUSTOM for HandoverType

### ISSUE THREAD

**Issue by costero-e** [ga4gh-beacon/beacon-v2#83]
When I execute Beacon Verifier and the schema being verified has the generic CUSTOM in HandoverType, the Verifier complains about the schema because CUSTOM doesn't match the OntologyTerm.json nomenclature when looking at [beaconCommonComponents.json](https://raw.githubusercontent.com/ga4gh-beacon/beacon-v2/main/framework/json/common/beaconCommonComponents.json) reference.
I would suggest to add CUSTOM like an Ontology, with a fake Ontology Id part. For example:
```
"HandoverType": {
"$ref": "./ontologyTerm.json",
"description": "Handover type, as an Ontology_term object with CURIE syntax for the `id` value. Use `CUSTOM:000001` for the `id` when no ontology is available.",
"examples": [
{
"id": "EFO:0004157",
"label": "BAM format"
},
{
"id": "CUSTOM:000001",
"label": "download genomic variants in .pgxseg file format"
}
]
},
```

**Comment by costero-e** [comment]
Yes, nice example, thank you @mbaudis . That's what I had to do for our Beacon to pass the Verifier (change CUSTOM to an ontology nomenclature). So it is already obvious that CUSTOM has to be an ontology and not the word "CUSTOM" itself or it would be better to make a PR and change CUSTOM for CUSTOM:01 or something like that?

**Comment by mbaudis** [comment]
@costero-e please see https://github.com/ga4gh-beacon/beacon-v2/commit/11312cd0bd0374c31b4e7902609b365eed459c3c Also, IMO https://github.com/ga4gh-beacon/beacon-v2/pull/82 should be merged @jrambla ...

**Comment by costero-e** [comment]
Ok, I addressed all the fixes from **schema-urgent-fixes** branch in this PR #85. If @jrambla and @mbaudis can you review and accept the PR we can move on.
